SensorsAnalyticsSDK/pom.xml

```diff
@@ -26,14 +26,6 @@
         </license>
     </licenses>
 
-    <developers>
-        <developer>
-            <name>renqingyou</name>
-            <email>renqingyou@sensorsdata.cn</email>
-            <url>http://sensorsdata.cn</url>
-        </developer>
-    </developers>
-
```

Comment:
I don't think we've changed this sufficiently to claim to be authors, let's leave it in.

Reply:
SGTM, my reasoning was that we didn't want to direct to him for changes in our repository such as the release scripts etc.